src/libraries/System.Private.CoreLib/src/System/Globalization/DateTimeParse.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Avoid BOM changes for existing files

Reply:
Fixed by https://github.com/dotnet/runtime/pull/115507/commits/89425c2286f2fadfce596adb41b0ce93e7fa9bcb